tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -87,6 +87,7 @@ def test_pantsd_broken_pipe(self):
             self.assertNotIn("broken pipe", run.stderr_data.lower())
             checker.assert_started()
 
+    @pytest.mark.skip(reason="flaky")
```

Comment:
I realize now that I should have created an issue when I first encountered this, but too late now :/ 

This is the stack trace I preserved:
```
pants/java/nailgun_protocol.py", line 147, in write_chunk
739                     E   	    sock.sendall(chunk)
740                     E   	OSError: [Errno 88] Socket operation on non-socket
```